src/renderer/shared/api/network/lib/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { WellKnownChain } from '@substrate/connect';
+
+import type { ChainId, ChainOptions } from '@shared/core';
+import { Chains } from './types';
+
+export function getKnownChain(chainId: ChainId): WellKnownChain | undefined {
+  const KnownChains: Record<ChainId, WellKnownChain> = {
+    [Chains.POLKADOT]: WellKnownChain.polkadot,
+    [Chains.KUSAMA]: WellKnownChain.ksmcc3,
+  };
+
+  return KnownChains[chainId];
+}
+
+export function isPolkadot(chainName: string): boolean {
+  return chainName === 'Polkadot';
+}
+
+export function isKusama(chainName: string): boolean {
+  return chainName === 'Kusama';
+}
+
+export function isTestnet(chainOptions?: ChainOptions[]): boolean {
+  return Boolean(chainOptions?.includes('testnet'));
+}
+
+export function isNameWithNumber(chainName: string): boolean {
+  return /^[0-9]+/.test(chainName);
```

Comment:
regexp checks that name starts with number so maybe name the function `isNameStartsWithNumber`?